src/app/containers/FrontPageMain/index.test.jsx

```diff
@@ -1,11 +1,84 @@
 import React from 'react';
 import FrontPageMain from '.';
 import { shouldShallowMatchSnapshot } from '../../helpers/tests/testHelpers';
+import frontPageDataIgbo from '../../../../data/prod/igbo/frontpage';
 import frontPageDataPidgin from '../../../../data/prod/pidgin/frontpage';
+import frontPageDataYoruba from '../../../../data/prod/yoruba/frontpage';
+
+const igboConfig = {
+  brandName: 'BBC News Ìgbò',
+  translations: {
+    home: 'Akụkọ',
+  },
+};
+
+const pidginConfig = {
+  brandName: 'BBC News Pidgin',
+  translations: {
+    home: 'Home',
+  },
+};
+
+const yorubaConfig = {
+  brandName: 'BBC News Yorùbá',
+  translations: {
+    home: 'Ìròyìn',
+  },
+};
+
+jest.mock('react', () => {
```

Comment:
Interesting approach to mock use context instead of wrapping the component in a ServiceContext.Provider with fake config, curious of others opinions of this method

Reply:
I'm definitely in favour of this approach (in fact [I've used it before](https://github.com/bbc/simorgh/blob/latest/src/app/containers/FrontPageSection/index.test.jsx#L147)). This is needed to make shallow renders work (simply wrapping the component in a context provider will only render the context provider; which is less than useless).
You could avoid it by doing deep renders; but I also don't like snapshot tests which are so brittle that they break on updates in other components/containers/psammead package bumps.

Reply:
Awesome, sounds great :)